pkg/vm/memory/memory.go

```diff
@@ -24,45 +24,43 @@ type ValidationRule func(*Memory, Relocatable) ([]Relocatable, error)
 
 // Memory represents the Cairo VM's memory.
 type Memory struct {
-	data                map[Relocatable]MaybeRelocatable
-	num_segments        uint
-	validation_rules    map[uint]ValidationRule
-	validated_addresses AddressSet
+	Data              map[Relocatable]MaybeRelocatable
+	NumSegments       uint
+	ValidationRules   map[uint]ValidationRule
+	ValidatedAdresses AddressSet
```

Comment:
Dont do this

Reply:
this is from the PR of range check i guess, i had to merge that branch into this one as i needed somethings